src/xtgeo/grid3d/_ecl_grid.py

```diff
@@ -67,26 +72,22 @@ class GridRelative(Enum):
     ORIGIN = auto()
 
     def to_grdecl(self) -> str:
-        if self == GridRelative.MAP:
-            return "MAP"
-        else:
-            return ""
+        return "MAP" if self == GridRelative.MAP else ""
```

Comment:
Looks buggy, why not?

```suggestion
        return "MAP" if self == GridRelative.MAP else "ORIGIN"
```